Dockerfile.testing

```diff
@@ -15,9 +12,14 @@ ADD dependencies /
 RUN ["chmod", "+x", "dependencies"]
 RUN /dependencies
 
-ADD install /
-RUN ["chmod", "+x", "install"]
-RUN /install
+# Requires building the testing image of stellar-core. See Makefile.
+RUN apt-get -y install libunwind8 postgresql curl sqlite iproute2 libc++abi1-10 libc++1-10
+COPY --from=stellar/stellar-core:testing /usr/local/bin/stellar-core /usr/bin/stellar-core
+
+# Requires building the dev image of stellar-horizon. See Makefile.
+COPY --from=stellar/stellar-horizon:dev /horizon /usr/bin/stellar-horizon
+
+RUN adduser --system --group --quiet --home /var/lib/stellar --disabled-password --shell /bin/bash stellar;
```

Comment:
❓ what is this line for? is this for core?

Reply:
When the stellar-core deb installs it has a post install step that sets up a stellar user if one doesn't exist. We need to do that ourselves if not using it.